src/pyscipopt/scip.pyx

```diff
@@ -2757,6 +2759,27 @@ cdef class Model:
 
         return infeasible, objective
 
+    def getBendersSubproblem(self, probnumber, Benders benders = None):
+        """Returns a Model object that wraps around the SCIP instance of the subproblem.
+        NOTE: This Model object is just a place holder and SCIP instance will not be freed when the object is destroyed.
+
+        Keyword arguments:
+        probnumber -- the problem number for subproblem that is required
+        benders -- the Benders' decomposition object for the that the subproblem belongs to (Default = None)
+        """
+        cdef SCIP_BENDERS* scip_benders
+        cdef SCIP* scip_subprob
+
+        if benders is None:
+            scip_benders = SCIPfindBenders(self._scip, "default")
+        else:
+            n = str_conversion(benders.name)
+            scip_benders = SCIPfindBenders(self._scip, n)
```

Comment:
I guess this question is for @stephenjmaher 

Why does `benders` doesn't know itself and always needs to find itself? instead of storing the name one could just store the C object, right?

Reply:
Is it possible to store the C object? I didn't see where we do this in the other plugins. Would it be something like
```
cdef SCIP_BENDERS* _benders
```
in benders.pxi?

Reply:
I guess... and then set it in the include? Oh yeah, but you are right, we are not doing this on the other plugins... maybe we should. Probably would be good here since SCIPfindBenders is called for everything

Reply:
I just tested this, it seems to work. I will make the changes now.